pep/app/pods/components/document/text/component.ts

```diff
@@ -32,52 +35,34 @@ export default class DocumentText extends Component<DocumentTextArgs> {
         this.parseDocumentText(args.text);
     }
 
+    loadXSLT() {
+        let request = new XMLHttpRequest();
+        request.open('GET', '/xmlToHtml.xslt', false);
+        request.send('');
+        return request.responseXML;
+    }
+
     @dontRunInFastboot
-    parseDocumentText(text: string) {
-        const XML = parseXML(text);
-        if (!(XML instanceof Error)) {
-            const artinfo = findElement(XML, PepXmlTagNames.ARTICLE_INFO);
-            artinfo.parentNode?.removeChild(artinfo);
+    async parseDocumentText(text: string) {
+        const xml = parseXML(text);
+
+        if (!(xml instanceof Error)) {
+            const xslt = await this.loadXSLT();
 
-            this.convertBibliography(XML);
-            this.convertPageBreaks(XML);
-            this.xml = XML;
+            if (xslt && document.implementation && document.implementation.createDocument) {
+                const processor = new XSLTProcessor();
+                const colors = this.theme.currentTheme.colors.links;
+                processor.setParameter('', 'glossaryColor', colors.glossary);
```

Comment:
this looks like the one spot where those colors are actually needed in-JS,  I don't see where that `glossaryColor` param is actually used in the XSLT file, and we are just setting colors with generated CSS/classes anyway, unless I'm missing something?